internal/delivery/film/film.go

```diff
@@ -0,0 +1,75 @@
+package film
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+
+	"github.com/dranikpg/dto-mapper"
+	"github.com/go-park-mail-ru/2023_1_ContentDealers/internal/domain"
+	"github.com/go-park-mail-ru/2023_1_ContentDealers/pkg/logging"
+
+	"github.com/gorilla/mux"
+)
+
+type Handler struct {
+	useCase UseCase
+	logger  logging.Logger
+}
+
+func NewHandler(useCase UseCase, logger logging.Logger) Handler {
+	return Handler{useCase: useCase, logger: logger}
+}
+
+func (h *Handler) GetByContentID(w http.ResponseWriter, r *http.Request) {
+	defer r.Body.Close()
+
+	idRaw := mux.Vars(r)["content_id"]
+	var id uint64
+
+	_, err := fmt.Sscanf(idRaw, "%d", &id)
```

Comment:
```suggestion
	id, ok := idRaw.(uint64)
```
вроде более каноничный способ